ironweb.d.ts

```diff
@@ -30,6 +30,7 @@ export interface GroupCreateOptions {
     groupID?: string;
     groupName?: string;
     addAsMember?: boolean;
+    userList?: string[];
```

Comment:
This should be called `memberList` as the current name doesn't make it clear what the users are. Since members and admins are both "users", we need to be specific about what list this is.